types/lockfile-lint-api/yarn.test.lock

```diff
@@ -0,0 +1,41 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Your test doesn’t actually _execute_, so you should remove this file. The test is just to validate the types that arise from using the library.

Reply:
Thanks, its completely obsolete then, as is the OTHER_FILES.txt. I removed both. Thanks!